### PR TITLE
Completed DOM Events exercise

### PR DESCRIPTION
**Pull request by maryalenn**
dom-events

### CODE REVIEW

File: dom-events/buttons.js [hunk 1]
```diff
@@ -0,0 +1,29 @@
+
+// CLICK EVENT
+function handleClick (event) {
+  console.log('button clicked');
+  console.log(event);
+  console.log(event.target);
+  var buttonClicked = document.querySelector('.click-button');
```
Comment (BrettAlbright) [inline dom-events/buttons.js]: This should be done outside the function, and you want to attach the addEventListener to the variable with the reference.

**Review by BrettAlbright — CHANGES_REQUESTED** [review]
make sure to target the element outside of the function and to call the addEventListener on that variable

File: dom-events/buttons.js (hunk 1 above)
Comment (maryalenn) [reply dom-events/buttons.js]: image

image

**Review by BrettAlbright — APPROVED** [review]
👍🏼

**Pull request merged**